roles/php56/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+- name: Install REMI repo for Enterprise Linux 7
+  yum: name=http://rpms.famillecollet.com/enterprise/remi-release-7.rpm state=present
+
+- name: Enable REMI repo for Enterprise Linux 7
+  copy: src=remi.repo dest=/etc/yum.repos.d/remi.repo
+
+- name: Install PHP
+  yum: name=php state=present enablerepo=remi
+
+- name: Install PHP extensions
+  yum: name={{ item }} state=present enablerepo=remi
+  with_items:
+        - php-cli
+        - php-bcmath
+        - php-mbstring
+        - php-mcrypt
+        - php-pdo
+        - php-mysql
+        - php-gmp
```

Comment:
`gmp` should not be needed, `bcmath`, `pdo` and `mysql` are not needed now, may be in the future.